### ISSUE TITLE
How can I get XKB engines to show in the language list?

### ISSUE THREAD

**Issue by bmansurov** [ibus/ibus-m17n#8]
Since KBD engines were blocked in [289c8b7](https://github.com/ibus/ibus-m17n/commit/289c8b738db5d424b796f0a9f79688d73b22280a), how do I get to show XKB engines in the list of languages (in IBUS prefernces -> Input method -> Add)?

For example, the 'uz' layout doesn't show in the list, even though it exists in `/usr/share/X11/xkb/symbols/uz`.

**Comment by pnemade** [comment]
I can see uz layouts available for selection in IBus

**Comment by bmansurov** [comment]
@pnemade what distribution are you using? I'm on Arch Linux. Anything extra you did besides installing ibus-m17n?

**Comment by pnemade** [comment]
I am using Fedora 29 Linux distribution. I have not done anything extra. I just selected Country name and then the presented input sources list already having both "Uzbek" and "Uzbek (Latin)" from that uz symbols file.

**Comment by bmansurov** [comment]
According to https://pkgs.org/download/ibus-m17n, Fedora 29 uses the version 1.3.4-27, while Arch Linux uses 1.4.1-1. I also had Uzbek (at least kbd version) working with the earlier version of ibus-m17n, but not anymore with the newer version.

**Comment by njapke** [comment]
I have had the same problem with a Kazakh keyboard layout (see #6). The problem is that the XKB layouts are not correctly referenced in `/usr/share/ibus/component/simple.xml`, which is a file from the normal ibus project (and under Arch Linux it is part of the ibus package). As a quick fix, I just added
```
<engine>
	<name>xkb:kk::kz</name>
	<language>kk</language>
	<license>GPL</license>
	<author>Peng Huang &lt;shawn.p.huang@gmail.com&gt;</author>
	<layout>kz</layout>
	<longname>Kazakh</longname>
	<description>Kazakh (Cyrillic)</description>
        <icon>ibus-keyboard</icon>
	<rank>99</rank>
</engine>
```
into my `simple.xml` and moved on, but to properly address the issue, the missing XKB layouts should be added to `simple.xml` of the ibus project. The problem only became apparent now that the KBD layouts have been removed from ibus-m17n. I believe my snippet from above could be easily adapted for other XKB layouts like Uzbek. Then, these layouts would also be accessible from regular ibus.
